### ISSUE TITLE
lightGBM model 테스트 결과

### ISSUE THREAD

**Issue by leejj5645** [kookmin-sw/2019-cap1-2019_3#11]
xgboost와 같은 gradient boosting tree model 인 lightGBM으로 데이터셋을 테스트
xgboost와 달리 댓글을 char 단위로 2에서 6 gram 생성, word 단위로 2에서 3 gram을 생성하여 
tfidfvectorizer로 벡터화 하였다. 벡터화한 데이터는 sparse matrix로 저장되고 fit_tranform()으로 lightGBM에 맞는 데이터 형태로 변경한 뒤  lightGBM의 파라미터 objective: 'multiclasss', num_class: '6'로  learning_rate는 0.2, 0.1, 0.05, 0.02, 0.01 으로 각각 학습시키고  L1 L2 등과 같은 규제와 num_leaves : 31, num_boost_round : 100, 200 ,300 으로 early stop 100을
 적용하여 학습시킨 결과 정확도가 65% 정도로 xgboost가 75%인데에 비해 낮은 결과를 보여 기존 xgboost 모델을 사용하기로 했다
